classes/class-wc-klarna-refund-fee.php

```diff
@@ -399,6 +411,22 @@ private function should_return_fee_info_be_shown() {
 
 		return false;
 	}
+
+	/**
+	 * Check if the store base country supports return fees.
+	 *
+	 * @return bool True if supported, false otherwise.
+	 */
+	private function is_return_fee_supported_country() {
+		$allowed_countries = array( 'AT', 'DE', 'DK', 'FI', 'FR', 'NL', 'NO', 'SE' );
+		$store_country     = WC()->countries->get_base_country();
```

Comment:
Question: Is this supposed to be based on the store country and not the country for the order?